dnsimple/dnsimple-developer
# New homepage

This PR moves the previous homepage to "Getting started" and replaces the content with quick access to specific API languages and feature announcements related to https://github.com/dnsimple/dnsimple-marketing/issues/544.

## QA 
Use https://deploy-preview-429--dnsimple-developer.netlify.app/ to browse around and make sure if UTM codes are correct for external links to the app and blog.

![Screenshot 2022-10-04 at 13-28-16 Screenshot](https://user-images.githubusercontent.com/80610/193808485-6452c421-8cca-4352-abb2-f92a42a447f1.png)
